packages/api/src/command/medical/admin/populate-fhir.ts

```diff
@@ -21,6 +21,9 @@ const { log } = Util.out("populateFhirServer");
 
 export type PopulateFhirServerResponse = { patientsOK: number; patientsError: number };
 
+/**
+ * @deprecated
+ */
 export async function populateFhirServer({
```

Comment:
I dont believe we use this anymore

Reply:
Not to be nit-picky, but why doesn't it handle multiple HIEs? It calls `queryDocumentsAcrossHIEs` which calls both CW and CQ.

If it really doesn't work, I'd either ticket fixing it or would remove the code. Leaving like this will surely become something to slow us down when maintaining one of the functions it calls - it's deferring assessing this for later, and now at least we have good context on the changes and diff between CW/CQ.

Thoughts?

Reply:
So we had discussed this [here](https://metriport.slack.com/archives/C065BLRBUDQ/p1703717816790779) but the main caveat is this uses override feature that is only being used by CW. We had discussed redoing the CW function but decided it wasnt worth the time at the moment to extract this logic away. 

So with that in mind i think it makes sense to then just ticket this as something to assess furthur. What do you think?

It also doesnt seem like we use this much anymore as well